scripts/Dockerfile

```diff
@@ -2,17 +2,19 @@ FROM jderobot/base
 
 # Custom Robot Repository
 RUN mkdir -p /opt/jderobot && cd /opt/jderobot && \ 
-	git clone -b melodic-devel https://github.com/JdeRobot/CustomRobots.git
+	git clone -b melodic-devel https://github.com/JdeRobot/CustomRobots.git && \
+	git clone https://github.com/JdeRobot/IndustrialRobots.git
```

Comment:
It seems that adding `-b web-template` is necessary. If the industrial robot exercise is still not working properly for RADI2.3. All modifications in this file and other RADI2.3 related files should be recovered.